Server/VikingVault.DataAccess/VikingVaultDbContext.cs

```diff
@@ -12,7 +12,15 @@ public class VikingVaultDbContext: DbContext
         public VikingVaultDbContext(DbContextOptions<VikingVaultDbContext> options)
             : base(options)
         { }
+        
+        public DbSet<User> User { get; set; }
```

Comment:
Since it is a DbSet with by definition contains more than 1 element, this property should be pluralized.